packages/workbox-webpack-plugin/src/lib/read-file-wrapper.js

```diff
@@ -15,35 +15,16 @@
 */
 
 /**
- * Use the `compiler.inputFileSystem._readFile` method instead of `fs.readFile`,
- * `readFile` is configured to use `compiler.inputFileSystem._readFile` during
- * the run phase of the webpack compilation lifecycle by passing the function
- * to the `setReadFile` function.
+ * A wrapper that calls readFileFn and returns a promise for the contents of
```

Comment:
Could you explain what readFileFn is it - I'm assuming it's a webpack provided function?

Reply:
Done.